build/common.project.props

```diff
@@ -84,6 +84,7 @@
     <Features>strict</Features>
     <!-- Same as SDK default, but without CandidateAssemblyFiles in front, which would search in Content items -->
     <AssemblySearchPaths>{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
+    <LangVersion>9</LangVersion>
```

Comment:
Can we use [`latestMajor`] (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#c-language-version-reference) instead of hardcoding `version 9`?

Reply:
I wouldn't do that since we don't control our CI machines and the cadence at which the build systems are updated there. 

I think that might be an option if we're ever on arcade.